<!-- path recorder launch -->
<launch >
	<!-- args -->
	<arg name="map_frame" default="/map" />
	<arg name="base_link_frame" default="/base_link" />
	<arg name="output_path_topic_name" default="/recorded_path" />
	<arg name="dist_interval" default="0.5" />
	<arg name="angle_interval" default="10.0" />

	<!-- path recorder -->
	<node name="path_recorder" pkg="path_follower" type="path_recorder" output="screen" >
		<param name="/map_frame" value="$(arg map_frame)" type="string" />
		<param name="/base_link_frame" value="$(arg base_link_frame)" type="string" />
		<param name="/output_path_topic_name" value="$(arg output_path_topic_name)" type="string" />
		<param name="/dist_interval" value="$(arg dist_interval)" type="double" />
		<param name="/angle_interval" value="$(arg angle_interval)" type="double" />
	</node >
</launch >
